packages/liveblocks-react/src/liveblocks.tsx

```diff
@@ -144,49 +144,57 @@ export function createLiveblocksContext<
     return;
   }
 
+  function useInboxNotificationsSelectorCallback(
+    state: CacheState<BaseMetadata>
+  ): InboxNotificationsState {
```

Comment:
@GuillaumeSalles Any particular reason why some selectors are stored as callback and others not? I know it is slightly better for performance to store selectors outside of component  (if possible) because `useCallback` has some performance cost. Curious if there are other reasons too

Reply:
I only used `useCallback` if necessary. If the function need to access any local variable, we're using `useCallback`, if not, it's a pure function outside of the hook. It's to be a lean a possible, not for performance.